Sources/ApiVideoPlayer/Views/ActionBarView.swift

```diff
@@ -265,4 +267,17 @@ public protocol ActionBarViewDelegate: AnyObject {
     func sliderValueChangedDidStop(position: Float64)
 }
 
+extension UIButton {
```

Comment:
Could you move the extension to https://github.com/apivideo/api.video-swift-player/tree/main/Sources/ApiVideoPlayer/Extensions?